basic/src/main/java/com/okta/spring/example/CodeFlowExampleApplication.java

```diff
@@ -53,9 +53,15 @@ public ExampleSecurityConfigurerAdapter(ApplicationContext applicationContext, O
 
         @Override
 		protected void configure(HttpSecurity http) throws Exception {
-        	super.configure(http);
+
+        	// In this example we allow anonymous access to the root index page
```

Comment:
Wow, I had white space viewing turned off in my editor.  I had a auto convert turned on a while ago, not sure what I changed.  Since this is an example I might just push another PR to fix that.